### ISSUE TITLE
Node order can not modified in inherited scene

### ISSUE THREAD

**Issue by alexzheng** [godotengine/godot#12139]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
godot 2.1

**Issue description:**

Move the Button up, and save.

<img width="243" alt="screen shot 2017-10-16 at 10 52 22 am" src="https://user-images.githubusercontent.com/1130047/31593311-76ab9880-b260-11e7-8f3f-dccf5759718c.png">

reopen the scene, the Button is still below the actions.

<img width="232" alt="screen shot 2017-10-16 at 10 51 57 am" src="https://user-images.githubusercontent.com/1130047/31593310-7670441a-b260-11e7-97a7-24d6dfe9dcc1.png">

**Steps to reproduce:**

**Link to minimal example project:**

**Comment by akien-mga** [comment]
This seems to be still valid in the current master branch.

**Comment by KoBeWi** [comment]
Works fine in 3.2

**Comment by Keetz** [comment]
@KoBeWi I just tested this in 3.2.2-stable and it is still an issue.

I created a scene Main.tscn.
Then i created a scene Child.tscn and instanced a few Node2D nodes.

I instanced the Child.tscn in the Main.tscn and enabled editable children.
I then added a new Node2D in the Main.tscn but as a child of the instanced Child.tscn and moved that node up between some of the other nodes.
Saved, closed and re-opened the Main.tscn and the node is at the bottom.

Step 1:
![image](https://user-images.githubusercontent.com/6707330/87636346-ead6c700-c740-11ea-8913-3d9d9772e8d6.png)

Step 2:
![image](https://user-images.githubusercontent.com/6707330/87636388-f88c4c80-c740-11ea-99c1-2a59e5ff886f.png)

Step 3:
![image](https://user-images.githubusercontent.com/6707330/87636401-fe822d80-c740-11ea-8389-5ece79f03b07.png)
